pyproject.toml

```diff
@@ -75,6 +75,27 @@ Changelog = "https://cotainr.readthedocs.io/en/stable/release_notes/index.html"
 [tool.hatch.version]
 path = "cotainr/__init__.py"
 
+[tool.hatch.build.targets.sdist]
+include = [
+    "/cotainr",
+    "/doc",
+    "README.md",
```

Comment:
I know realize that the content of  README.md is included at the bottom of the PKG-INFO file in the sdist, so maybe we should leave out the README.md file.
```suggestion
```